src/js/register.jsx

```diff
@@ -67,65 +89,76 @@ class Register extends React.Component {
                   autoComplete="off"
                   className="form-control"
                   id="email"
-                  onChange={(e) => this.setState({ email: e.target.value })}
+                  onChange={(e) => this.setState({ email: e.target.value ,
+                                                   userId: null})}
+                  onBlur={() => this.fetchUserIfExists()}
                   placeholder="Email"
                   type="email"
                   value={this.state.email}
                 />
               </div>
-              <div className="form-group">
-                <label htmlFor="password">Enter your password</label>
-                <input
-                  autoComplete="off"
-                  className="form-control"
-                  id="password"
-                  onChange={(e) => this.setState({ password: e.target.value })}
-                  placeholder="Password"
-                  type="password"
-                  value={this.state.password}
-                />
-              </div>
-              <div className="form-group">
-                <label htmlFor="password-confirmation">Confirm your password</label>
-                <input
-                  autoComplete="off"
-                  className="form-control"
-                  id="password-confirmation"
-                  onChange={(e) => this.setState({ passwordConfirmation: e.target.value })}
-                  placeholder="Password confirmation"
-                  type="password"
-                  value={this.state.passwordConfirmation}
-                />
-              </div>
-              <div className="form-check">
-                <input
-                  checked={this.state.acceptTOS}
-                  className="form-check-input"
-                  id="tos-check"
-                  onChange={() => this.setState({ acceptTOS: !this.state.acceptTOS })}
-                  type="checkbox"
-                />
-                <label className="form-check-label" htmlFor="tos-check">
-                  I agree to the{" "}
-                  <a href="/terms-of-service" target="_blank">
-                    Terms of Service
-                  </a>
-                  .
-                </label>
-              </div>
-              <div className="form-check mb-3">
-                <input
-                  checked={this.state.acceptDataTOS}
-                  className="form-check-input"
-                  id="data-tos-check"
-                  onChange={() => this.setState({ acceptDataTOS: !this.state.acceptDataTOS })}
-                  type="checkbox"
-                />
-              </div>
-              <button className="btn btn-lightgreen float-right mb-2" type="submit">
-                Register
-              </button>
-            </form>
+              {this.state.userId === false && 
+               <React.Fragment>
```

Comment:
we can replace `<React.Fragment>` for just `<>` and `</React.Fragment>` for `</>`